oops/oops_test.go

```diff
@@ -70,6 +71,23 @@ func doubleWrapf() error {
 	return oops.Wrapf(oops.Wrapf(oops.Wrapf(rootCause, "yuck"), "bad"), "why would you do this")
 }
 
+func stripPathPrefix(s string) string {
+	// Strip path prefixes from filenames, aids comparisons in stack trace tests
+	_, filename, _, ok := runtime.Caller(0)
+	if !ok {
+		panic("Cant work out where we are")
+	}
+	pth := path.Dir(filename)
+	return strings.ReplaceAll(s, pth, "github.com/samsarahq/go/oops")
+
+}
+
+func fixLineNumbers(s string) string {
+	// Sanitize line numbers in stack traces to avoid cat and mouse
```

Comment:
Is it worth elaborating a bit on the exact intent here and what is meant by "cat and mouse," perhaps with an example? In my experience, regular expressions -- especially those whose intent is not self evident to future readers/maintainers -- can be implicated in future bugs. Could a clarifying code comment help?